tar/tarsnap.1-mdoc.in

```diff
@@ -642,6 +646,11 @@ option is also specified.
 (c and d modes only)
 Print statistics for the archive being created (c mode) or delete (d mode).
 See "PRINTING ARCHIVE STATISTICS" below for information on the output format.
+.It Fl -progress-bytes Ar X
+Display a progress message (as is generated from SIGUSR1 or SIGINFO) after
+processing each
+.Ar X
+bytes.
```

Comment:
This isn't quite correct.

    --progress-bytes 100k

prints a message after processing a file if tarsnap has processed at least 100k bytes more than the last time it gave a message.  So:
- it doesn't give multiple messages while uploading a large file.
- the "total so far" isn't going to increase by exactly 100k each file.

I couldn't find a succinct way of stating that, so I figured the current man-page text was good enough.